app/models/search_settings.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: search_settings
+#
+#  id           :integer          not null, primary key
+#  community_id :integer          not null
+#  main_search  :string(255)      default("KEYWORD"), not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+
+class SearchSettings < ActiveRecord::Base
+  attr_accessible(
```

Comment:
With the new RoR is attr_accessible anymore needed? These should be validated in controller level.